src/bafrio.f90

```diff
@@ -138,7 +138,7 @@ SUBROUTINE BAFRINDEXL(LU,IB,LX,IX)
 ! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
       END SUBROUTINE BAFRINDEXL
 
-!> This subprogram is calling bafread to reas an unformatted fortran
+!> This subprogram calls bafread() to reas an unformatted fortran
 !> record. The difference between bafrread() and bafrreadl() is the
 !> kind type of intergers in the argument list.
```

Comment:
integers*

Reply:
Awkward phrasing but not something I'm going to worry about. I think in time we'll be able to eliminate this code duplication.